pandas/_libs/tslibs/timestamps.pyi

```diff
@@ -211,3 +215,7 @@ class Timestamp(datetime):
     ) -> np.datetime64: ...
     @property
     def _date_repr(self) -> str: ...
+    @property
+    def days_in_month(self) -> int: ...
+    @property
+    def daysinmonth(self) -> int: ...
```

Comment:
in stubs could these be defined as aliases or do they need to be declared separately?

Reply:
They have to be declared separately in stubs, because the type checkers won't see that it is a property if you put the alias in the stubs.